stacks/syrah/templates/modules.yaml.j2

```diff
@@ -131,6 +131,10 @@ modules:
           set:
             HDF5_CC: h5pcc
             HDF5_FC: h5pfc
+      intel-oneapi-compilers-classic:
+        environment:
+          set:
+            INTEL_ROOT: ${PREFIX}
```

Comment:
`module show intel` show INTEL_ROOT already in the module

Reply:
Yes, indeed that already happend in the module we've created, not in the module spack creates.